README.md

```diff
@@ -32,6 +34,8 @@ Don't edit the `public/` folder directly! Jekyll will overwrite everything when
 - To build the static site, run `rake build`.
 - To build and serve the app, run `rake serve`.
 - To deploy to 18F's cloud, run `rake deploy`.
+- To add dummy data for developing against, run `rake populate:manifests`.
+- To (re)build the Elasticsearch index, run `rake search:index`.
```

Comment:
so many steps! just created a card for automating all of this by putting it in a script https://trello.com/c/RieEsKeK